profiles/Tier30/T30_Warrior_Arms.simc

```diff
@@ -108,24 +108,27 @@ actions.hac+=/wrecking_throw
 actions.single_target=sweeping_strikes,if=spell_targets.whirlwind>1
 actions.single_target+=/mortal_strike
 actions.single_target+=/rend,if=remains<=gcd|talent.tide_of_blood&cooldown.skullsplitter.remains<=gcd&(cooldown.colossus_smash.remains<=gcd|debuff.colossus_smash.up)&dot.rend.remains<dot.rend.duration*0.85
-actions.single_target+=/avatar,if=talent.warlords_torment&rage.pct<33&(cooldown.colossus_smash.ready|debuff.colossus_smash.up|buff.test_of_might.up)|!talent.warlords_torment&(cooldown.colossus_smash.ready|debuff.colossus_smash.up)
+actions.single_target+=/avatar,if=cooldown.colossus_smash.ready|debuff.colossus_smash.up|buff.test_of_might.up
 actions.single_target+=/spear_of_bastion,if=cooldown.colossus_smash.remains<=gcd|cooldown.warbreaker.remains<=gcd
 actions.single_target+=/warbreaker
 actions.single_target+=/colossus_smash
 actions.single_target+=/thunderous_roar,if=buff.test_of_might.up|talent.test_of_might&debuff.colossus_smash.up&rage.pct<33|!talent.test_of_might&debuff.colossus_smash.up
 actions.single_target+=/bladestorm,if=talent.hurricane&(buff.test_of_might.up|!talent.test_of_might&debuff.colossus_smash.up)|talent.unhinged&(buff.test_of_might.up|!talent.test_of_might&debuff.colossus_smash.up)
 actions.single_target+=/skullsplitter,if=talent.tide_of_blood&dot.rend.remains&(debuff.colossus_smash.up|cooldown.colossus_smash.remains>gcd*4&buff.test_of_might.up|!talent.test_of_might&cooldown.colossus_smash.remains>gcd*4)|rage<30
-actions.single_target+=/execute,if=buff.sudden_death.react
+actions.single_target+=/execute,if=buff.sudden_death.react&debuff.colossus_smash.down
 actions.single_target+=/shockwave,if=talent.sonic_boom.enabled
 actions.single_target+=/ignore_pain,if=talent.anger_management|talent.test_of_might&debuff.colossus_smash.up
-actions.single_target+=/whirlwind,if=talent.storm_of_swords&talent.battlelord&rage.pct>80&debuff.colossus_smash.up
+actions.single_target+=/whirlwind,if=talent.storm_of_swords&talent.battlelord&debuff.colossus_smash.up
+actions.single_target+=/execute,if=buff.sudden_death.react
 actions.single_target+=/overpower,if=charges=2&!talent.battlelord&(debuff.colossus_smash.down|rage.pct<25)|talent.battlelord
 actions.single_target+=/whirlwind,if=talent.storm_of_swords|talent.fervor_of_battle&active_enemies>1
 actions.single_target+=/thunder_clap,if=talent.battlelord&talent.blood_and_thunder
 actions.single_target+=/overpower,if=debuff.colossus_smash.down&rage.pct<50&!talent.battlelord|rage.pct<25
 actions.single_target+=/whirlwind,if=buff.merciless_bonegrinder.up
 actions.single_target+=/cleave,if=set_bonus.tier29_2pc&!talent.crushing_force
-actions.single_target+=/slam,if=rage>30&(!talent.fervor_of_battle|talent.fervor_of_battle&active_enemies=1)
+actions.single_target+=/whirlwind,if=rage.pct>90
+actions.single_target+=/rend,if=rage.pct>90
```

Comment:
slam overall is inefficient damage per rage spent. I've removed it to instead press a damage / rage spent button (rend and WW)

Reply:
As per Discord convo: Add conditionals for use with off-meta builds, rather than removing the line entirely - while it isn't an important part of the current rotation, not everyone will always use it and its important to be able to quantify various build performance.

Reply:
Done